File: spring-cloud-integrated-demo/membership-service/src/main/scala/com/github/jccode/springcloud/integrateddemo/membership/Application.scala
package com.github.jccode.springcloud.integrateddemo.membership

import com.github.jccode.springcloud.integrateddemo.common.config.MicroServiceConfig
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(Array(classOf[MicroServiceConfig]))
class Application {

}

object Application {
  def main(args: Array[String]): Unit = {
    SpringApplication.run(classOf[Application], args: _*)
  }
}

File: spring-cloud-integrated-demo/order-service/src/main/java/com/github/jccode/springcloud/integrateddemo/order2/Application.java
package com.github.jccode.springcloud.integrateddemo.order2;

import com.github.jccode.springcloud.integrateddemo.common.config.MicroServiceConfig;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;

@SpringBootApplication
@Import(MicroServiceConfig.class)
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}
